core/shared/src/main/scala/zio/clock/package.scala

```diff
@@ -79,6 +81,9 @@ package object clock {
   val currentDateTime: ZIO[Clock, DateTimeException, OffsetDateTime] =
     ZIO.accessM(_.get.currentDateTime)
 
+  val instant: ZIO[Clock, Nothing, java.time.Instant] =
```

Comment:
Probably should add `localDateTime` as an accessor as well. I can also pick up in a follow up.